python/tvm/topi/nn/conv2d.py

```diff
@@ -424,17 +424,15 @@ def conv2d_NCHWc(data, kernel, stride, padding, dilation, layout, out_layout, ou
 
     return te.compute(
         oshape,
-        lambda n, oc_chunk, oh, ow, oc_block: te.sum(
+        lambda n, occ, oh, ow, oc_block: te.sum(
```

Comment:
Could we preserve a group=1 code path like `conv2d_NCHWc_int8` below?

Reply:
Done, please take a look again.